File: UAUTitanium/UAUMobileWebAPI/UAUMobileWebAPI/Models/Usuario.vb
Namespace Models.Validador
    Public Class Usuario
        Public Property Codigo() As Integer
        Public Property Nome() As String
        Public Property Cpf() As String
        Public Property DataNasc() As Object
            Get
                Return dtnasc_value
            End Get
            Set(value As Object)
                Try
                    If Not IsDBNull(value) Then
                        Dim dt = CType(value, DateTime)
                        dtnasc_value = dt.ToString("yyyy-MM-dd HH:mm:ss")
                    End If
                Catch ex As Exception
                    Throw
                End Try
            End Set
        End Property
        Private dtnasc_value As String
        Public Property Email() As String
        Public Property Endereco() As String
        Public Property Numero() As String
        Public Property Referencia() As String
        Public Property Bairro() As String
        Public Property Cep() As String
        Public Property Cidade() As String
        Public Property UF() As String
        Public Property Login() As String
        Public Property UsuarioUAU() As String
    End Class
End Namespace

File: UAUTitanium/Componentes Mobile/widgets/Login/models/Usuario.js
/** @class widgets.Login.models.Usuario
 * Modelo que representa as informações do usuário.
 * @private
 * @alteracao 21/01/2015 176562 Projeto Carlos Eduardo Santos Alves Domingos
 * Criação.
 * @cfg {Number} Codigo Código do usuario.
 * @cfg {String} Nome Nome do usuário.
 * @cfg {String} Cpf CPF do usuario
 * @cfg {Date} DataNasc Data de nascimento.
 * @cfg {String} Email Email do usuário.
 * @cfg {String} Endereco Endereço do usuario.
 * @cfg {String} Numero Número do endereço do usuario.
 * @cfg {String} Referencia Ponto de referência para o endereço do usuario.
 * @cfg {String} Bairro Bairro.
 * @cfg {String} Cep Cep.
 * @cfg {String} Cidade Cidade.
 * @cfg {String} UF UF.
 * @cfg {String} Login Login utilizado no UAU web.
 */
exports.definition = {
	config: {
		columns: {
		    "Codigo": "integer PRIMARY KEY",
		    "Nome": "varchar",
		    "Cpf": "varchar",
		    "DataNasc": "datetime",
		    "Email": "varchar",
		    "Endereco": "varchar",
		    "Numero": "varchar",
		    "Referencia": "varchar",
		    "Bairro": "varchar",
		    "Cep": "varchar",
		    "Cidade": "varchar",
		    "UF": "varchar",
		    "Login": "varchar",
		    "UsuarioUAU": "varchar"
		},
		adapter: {
			type: "sql",
			collection_name: "Usuario"
		}
	},
	extendModel: function(Model) {
		_.extend(Model.prototype, {
			// extended functions and properties go here
		});

		return Model;
	},
	extendCollection: function(Collection) {
		_.extend(Collection.prototype, {
			// extended functions and properties go here
		});

		return Collection;
	}
};